src/Compilers/CSharp/Portable/Binder/Binder_Crefs.cs

```diff
@@ -92,6 +92,12 @@ private NamespaceOrTypeSymbol BindNamespaceOrTypeSymbolInCref(TypeSyntax syntax)
             NamespaceOrTypeSymbol namespaceOrTypeSymbol = BindNamespaceOrTypeSymbol(syntax, unusedDiagnostics);
             unusedDiagnostics.Free();
 
+            // BindNamespaceOrTypeSymbol will wrap any tuple types in a TupleTypeSymbol. We unwrap it here, as doc comments don't consider the (T,T) form of tuples
+            if(namespaceOrTypeSymbol is TupleTypeSymbol t)
```

Comment:
Will this handled nested cases: say when a tuple is a generic argument of a type?

Reply:
We don't support that case: you can't use types as a type parameter, only identifiers (we issue ```warning CS1658: Type parameter declaration must be an identifier not a type``` if you try).